freex/src/components/AddOffer/AddOffer/AddOffer.jsx

```diff
@@ -29,84 +51,81 @@ const AddOffer = () => {
       },
     };
 
-    addDoc(offersCollectionRef, offerData);
+    try {
+      addDoc(offersCollectionRef, offerData);
```

Comment:
Wydaje mi się, że to tak naprawdę nie działa, navigate i toast.success powinno być dopiero jak promise będzie resolved, czyli powinno być addDoc(offersCollectionRef, offerData).then(()=>navigate('/mojeoferty');
      toast.success('Dodano ofertę')) Wydaje mi się, że inaczej będzie się odpalało toast.success i navigate nawet jak nie uda się dodać dokumentu

Reply:
Masz rację! Później zmienię.